### PR TITLE
Updated client library docs

### PR DESCRIPTION
**Pull request by yuce**
## Overview

Updates Client Libraries section of the docs

## Pull request checklist

- [x] I have read the [contributing guide](https://github.com/pilosa/pilosa/blob/master/CONTRIBUTING.md).
- [x] I have agreed to the [Contributor License Agreement](https://cla-assistant.io/pilosa/pilosa).
- [x] I have updated the [documentation](https://github.com/pilosa/pilosa/tree/master/docs).
- [x] I have resolved any merge conflicts.
- [ ] I have included tests that cover my changes.
- [ ] All new and existing tests pass.

## Code review checklist
This is the checklist that the reviewer will follow while reviewing your pull request. You do not need to do anything with this checklist, but be aware of what the reviewer will be looking for.

- [ ] Ensure that any changes to external docs have been included in this pull request.
- [ ] If the changes require that minor/major versions need to be updated, tag the PR appropriately.
- [ ] Ensure the new code is [properly commented](https://github.com/golang/go/wiki/CodeReviewComments#doc-comments) and follows [Idiomatic Go](https://dmitri.shuralyov.com/idiomatic-go).
- [ ] Check that tests have been written and that they cover the new functionality.
- [ ] Run tests and ensure they pass.
- [ ] Build and run the code, performing any applicable integration testing.

### CODE REVIEW

File: docs/client-libraries.md [hunk 1]
```diff
@@ -13,220 +13,252 @@ nav = [
...
+        stargazer.bitmap(14),
+        stargazer.bitmap(19)
+    ),
+    language.bitmap(1)
+)
+mutually_starred = client.query(query).result.bitmap.bits
+print("User 14 or 19 starred, written in language 1:", mutually_starred)
+
+# Set user 99999 as a stargazer for repository 77777
+client.query(stargazer.setbit(99999, 77777))`
```
Comment (codysoyland) [inline docs/client-libraries.md]: This Python block is not properly closed. Needs 3 tick marks.

Comment (yuce) [reply docs/client-libraries.md]: Thanks @codysoyland, updated.

Comment (codysoyland) [reply docs/client-libraries.md]: No problem. Note there is still a single tick mark at the end of line 169.

Comment (yuce) [reply docs/client-libraries.md]: Updated, hopefully this time I got it right.

**Pull request merged**